axis_backend/apps/clients/services/client_service.py

```
@@ -0,0 +1,397 @@
+            client_id: Client ID to update
+            (other args same as create_client)
+
+        Returns:
+            Updated Client instance
+
+        Raises:
+            ValidationError: If validation fails
+        """
+        client = self.repository.get_by_id(client_id)
+        if not client:
+            raise ValidationError(f"Client with ID '{client_id}' not found")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)